ocaml/gencert/pem.ml

```diff
@@ -58,15 +60,21 @@ let key =
   key_footer kind *> return (String.concat "" [header kind; body; footer kind])
   <?> "key"
 
+let line = take_till is_eol *> end_of_line
+
+let until_key = fix (fun m -> key <|> line *> m) <?> "until_key"
```

Comment:
I was going to suggest to use `skip_many end_of_line`, but this should be more efficient. I think skip_many would skip to the end of the file and then backtrack until it finds something that succeeds parsing, whereas this one stops skipping whenever the key matches.